usr/src/cmd/pkgsrc-wrapper/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2023 MNX Cloud, Inc.
+#
+
+PROG=pkgsrc-wrapper
+#KSHPROG=pkgsrc-wrapper
+
+# Add programs to $WRAPPER_LINKS that you want to actually have hardlink to
+# the pkgsrc wrapper.
+WRAPPER_LINKS=csh
+WRAPPER_LINKS+=tcsh
+WRAPPER_LINKS+=zsh
+
+include ../Makefile.cmd
+
+.KEEP_STATE:
+
+all: $(PROG)
+
+install: all $(ROOTPROG) $(WRAPPER_LINKS)
+
+$(WRAPPER_LINKS):
```

Comment:
Honestly, I don't think this works, and the ONLY reason we get it in the PI is because of `manifest`.  Advice welcome.

Reply:
There are ROOTLINKS in $CMD/grep's Makefile that are probably better examples to follow.

Reply:
Much better now.